bridges/otelzerolog/go.mod

```diff
@@ -0,0 +1,23 @@
+module go.opentelemetry.io/contrib/bridges/otelzerolog
+
+go 1.22.1
```

Comment:
This repo currently needs to support Go 1.21.

Reply:
I resolved this in #5782